src/urllib3/response.py

```diff
@@ -154,7 +177,7 @@ class is also compatible with the Python standard library's :mod:`io`
         value of Content-Length header, if present. Otherwise, raise error.
     """
 
-    CONTENT_DECODERS = ['gzip', 'deflate']
+    CONTENT_DECODERS = ['gzip', 'deflate', 'br']
```

Comment:
We need to conditionally add `'br'` to `CONTENT_DECODERS`

Reply:
Yes, thank you